### PR TITLE
Add tpm policy generation code

### PR DESCRIPTION
**Pull request by mjg59**

### CODE REVIEW

File: app-crypt/tpmpolicy/tpmpolicy-20160404.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86 arm64"
+IUSE=""
+
+S="${WORKDIR}"
+
+src_install() {
+    insinto /usr/sbin
+    doins "${FILESDIR}"/tpm_hostpolicy.sh
```
Comment (marineam) [inline app-crypt/tpmpolicy/tpmpolicy-20160404.ebuild]: There is a `dosbin` shortcut for this which also ensures the right permission bits

File: app-crypt/tpmpolicy/files/tpm_hostpolicy.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+o -n '{'
```
Comment (marineam) [inline app-crypt/tpmpolicy/files/tpm_hostpolicy.sh]: Looks like copy/paste missed a bit here :)

File: app-crypt/tpmpolicy/tpmpolicy-20160404.ebuild (hunk 1 above)
Comment (marineam) [inline app-crypt/tpmpolicy/tpmpolicy-20160404.ebuild]: Also for a script that lands in the path I'd vote leave off the .sh

**Comment by mjg59** [comment]
Ok, fixed those issues

File: app-crypt/tpmpolicy/files/tpm_hostpolicy [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```
Comment (marineam) [inline app-crypt/tpmpolicy/files/tpm_hostpolicy]: I'd recommend a `set -e` so it aborts if it can't actually read the tpm and so on.

Comment (mjg59) [inline app-crypt/tpmpolicy/files/tpm_hostpolicy]: Done, along with an explicit TPM check and error message

**Comment by marineam** [comment]
LGTM

**Pull request merged**